### PR TITLE
feat: add description to drip lists (closes #642)

### LINKED ISSUE #642: Add Description to Drip Lists
MVP
- string only (no HTML/markdown)
- max 1000 characters (?)

make a new component to house editing the list, title, description to be used in Edit Drip List and Create

### PR DESCRIPTION
**Pull request by evvvritt**
closes #642 
- desc on create list
- desc on edit list
- expands more/less if longer than 2 lines

### CODE REVIEW

**Comment by evvvritt** [comment]
@efstajas (see my discord DM) but, not sure how to resolve the `svelte check`.
I extended the schema in v3.ts but it still doesn't see description there.
(it passes if i add description to v1.ts and v2.ts)
any ideas?

File: src/lib/components/drip-list-editor/drip-list-editor.svelte [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    <TextArea
+      bind:value={dripList.description}
+      resizable={true}
+      validationState={textAreaValidationState}
+    />
+  </FormField>
+
+  <div class:order-first={showListFirst}>
+    <FormField title="Members*">
+      <!-- TODO: This crashes when entering some non-valid github url -->
```
Comment (efstajas) [inline src/lib/components/drip-list-editor/drip-list-editor.svelte:63]: is that still a thing? 🤔

**Review by efstajas — APPROVED** [review]
Nice! Code changes LGTM 👍

File: src/lib/components/drip-list-editor/drip-list-editor.svelte (hunk 1 above)
Comment (efstajas) [reply src/lib/components/drip-list-editor/drip-list-editor.svelte:63]: @jtourkos could you clarify?

**Pull request merged**